architecture/additional_concepts/ephemeral-storage.adoc

```diff
@@ -0,0 +1,87 @@
+[[architecture-additional-concepts-ephemeral-storage]]
+= Ephemeral Local Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+*Note: this only applies if the ephemeral-storage tech preview in
+{product-title} 3.10 has been enabled.  This is disabled by default;
+to enable this features, please see
+xref:../install_config/configuring_ephemeral.adoc[Configuring for
+Ephemeral Storage].*
+
+== Overview
+
+In addition to persistent storage, pods and containers may require
+ephemeral (transient) local storage for their operation.  The lifetime
+of this ephemeral storage does not extend beyond the life of the
+individual pod, and this ephemeral storage cannot be shared across
+pods.
+
+Prior to {product-title} 3.10, ephemeral local storage was exposed to
+pods via the container’s writable layer, logs directory, and EmptyDir
```

Comment:
@sjenning the log directory is only really used if you run with cri-o since dockershim didnt do file based logging last i checked.  can you confirm?